controller_test.go

```diff
@@ -65,30 +72,30 @@ package main
 // 	}
 // }
 
-// func (f *fixture) newController() (*Controller, informers.SharedInformerFactory, kubeinformers.SharedInformerFactory) {
-// 	f.client = fake.NewSimpleClientset(f.objects...)
-// 	f.kubeclient = k8sfake.NewSimpleClientset(f.kubeobjects...)
+func (f *fixture) newController() (*Controller, informers.SharedInformerFactory, kubeinformers.SharedInformerFactory) {
```

Comment:
Why did you un-comment this if there is no test for it?

Reply:
upss, just a mistake